[TASK] Promote the GitLab project template

linawolf opened the pull request:
- Added GitLab as a third deployment option alongside AWS and platform.sh
- Linked directly to the GitLab TYPO3 template and to the related article on typo3.org
- Included GitLab logo image with consistent whitespace for visual alignment
- Improves developer onboarding with GitLab CI/CD integration

![image](https://github.com/user-attachments/assets/e1834bf3-ae57-4f8d-ac0f-65038457c7ad)

linawolf committed:
[TASK] Promote the GitLab project template

- Added GitLab as a third deployment option alongside AWS and platform.sh
- Linked directly to the GitLab TYPO3 template and to the related article on typo3.org
- Included GitLab logo image with consistent whitespace for visual alignment
- Improves developer onboarding with GitLab CI/CD integration

ochorocho commented on templates/default/root.html.twig:
```diff
@@ -257,6 +257,29 @@
+                    </div>
+                    <div class="card-footer">
+                        <a href="https://gitlab.com/gitlab-org/project-templates/typo3-distribution" class="btn btn-primary">
+                            <span class="btn-text">Use GitLab Template</span>
```
As a user i would exect the link "Use GitLab Template" to direct me directly to the closest possible page to actually use the Template:
* GitLab.com https://gitlab.com/projects/new#create_from_template
* Self hosted GitLab:  https://<your-domain>/projects/new#create_from_template

But somehow still keep the reference to the actual repository if possible.

linawolf replied:
I started with that link, however TYPO3 is then somewhere down the list which might be confusing?

ochorocho replied:
Well, it might be. :-) 

A short hint/note to the "Create from Template"-Page would be helpful.

linawolf replied:
Done.

ochorocho reviewed (commented):
LGTM 🚀

linawolf committed:
[TASK] Link to dedicated documentation pages for installation method

It is confusing to users that the "Installation" page only talks about
composer when they are currently trying to do a tar legacy installation.

Also who wants to contribute should be directed to the contribution guide not only to GitHub.

All links are only displayed if the page already existed for that version.

linawolf committed:
[TASK] Link to dedicated documentation pages for installation method

It is confusing to users that the "Installation" page only talks about
composer when they are currently trying to do a tar legacy installation.

Also who wants to contribute should be directed to the contribution guide not only to GitHub.

All links are only displayed if the page already existed for that version.

linawolf commented:
@andreaskienast this needs merging, it was was already approved by @ochorocho
